core/src/databases/table.rs

```diff
@@ -203,6 +220,22 @@ impl Table {
     }
 }
 
+impl ToNode for Table {
```

Comment:
I would do:

```
impl From<Table> for Node {
    fn from(t: Table) -> Self {
        ....
    }
}
```

Reply:
Allows you to have automatic conversion

Reply:
Very fine with that as well

Reply:
looks great  :-)

Reply:
Ok no real automatic conversion but def better

Reply:
I think if you make a function that accepts an `impl Into<Node>` instead of Node directly the conversion would be automatic

Reply:
not really, in that case a receive something that impl Into<Node> but I still have to call into() inside method body, so more or less the same